### PR TITLE
Source Intercom: Add tickets stream

### PR DESCRIPTION
**Pull request by krishan711**
## What

I've added the tickets stream so we can pull issues from intercom as shown in their API: https://developers.intercom.com/docs/references/rest-api/api.intercom.io/Tickets/ticket/

## How

I created a new stream called `tickets`.

## User Impact

Nothing changed in existing streams, just a new stream added.

## Can this PR be safely reverted and rolled back?

- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: airbyte-integrations/connectors/source-intercom/metadata.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: d8313939-3782-41b0-be29-b3ca20d8dd3a
-  dockerImageTag: 0.6.6
+  dockerImageTag: 0.6.7
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-intercom/metadata.yaml:13]: @krishan711 new streams are a minor update can you change to 0.7.0 and also bump the version in project.toml and run poetry lock? Thanks

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Comment by marcosmarxm** [comment]
If possible update the description of the contribution and link an existing issue or discussion.

**Comment by krishan711** [comment]
> If possible update the description of the contribution and link an existing issue or discussion.

Thanks, updated the description. please let me know what i should add more detail about if anything.

**Comment by marcosmarxm** [comment]
Running CI Tests

File: airbyte-integrations/connectors/source-intercom/source_intercom/manifest.yaml [hunk 2]
```diff
@@ -2377,6 +2377,560 @@ definitions:
...
+      name: "tickets"
+      path: "tickets/search"
+      data_field: "tickets"
+      page_size: 150
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        "$ref": "#/definitions/ticket"
+  ticket:
+    type: object
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-intercom/source_intercom/manifest.yaml:2400]: It misses `additionalProperties: true` to all object fields.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
One small detail

**Comment by marcosmarxm** [comment]
@krishan711 running CI, let's wait for the results.

**Comment by krishan711** [comment]
@marcosmarxm i cant tell what i need to do to make these checks pass. could i get some guidance here by any chance pls?

**Pull request closed without merging**